main.go

```diff
@@ -57,6 +82,25 @@ func Start() {
 		log.Fatalln(fmt.Errorf("cannot write resolv.conf file: %s", writeResolvErr).Error())
 	}
 
+	netConfig := path.Join(handlers.CNIConfDir, handlers.DefaultCNIConfFilename)
+	_, err = os.Stat(netConfig)
```

Comment:
these two lines should be able to "in-line" ?

Reply:
Yep...